web_responsive/static/src/js/web_responsive.js

```diff
@@ -535,8 +535,10 @@ odoo.define('web_responsive', function (require) {
         }),
 
         start: function () {
-            this.$btnMaximize = this.$('.o_maximize_btn');
-            this.$btnMinimize = this.$('.o_minimize_btn');
+            // Disable modal auto-focus: Workaround to not block edition mode
```

Comment:
```suggestion
            // Disable PDF modal auto-focus: Workaround to not block edition mode
```

Reply:
it should be then "document viewer", as you can preview also images

Reply:
Oh sorry I thought that the only one that stole focus was the PDF one.

TBH I never managed to reproduce the error this is fixing, so it's hard to review for me.

Reply:
Look at the TT for the bug description